### ISSUE TITLE
Please consider Angular 2's Renderer API

### ISSUE THREAD

**Issue by PatrickJS** [steveblue/ngus#5]
There's a better way

[react-native-renderer](https://github.com/angular/react-native-renderer)
[server_dom_renderer](https://github.com/angular/universal/blob/master/modules/server/src/server_dom_renderer.ts)
[angular2/src/render](https://github.com/angular/angular/tree/master/modules/angular2/src/render)

the api you're looking for is something like

``` typescript
@FamousComponent({
      tree: '<node id="foo" class="bar"></node>',
      behaviors: {
         '#foo': {
            'position':[250,50],
            'size': [100,100]
         },
         '.bar': {
            'style': {
              'background-color': 'red'
            }
         }
      }
})
export class FaMesh {

}
```

**Comment by steveblue** [comment]
@gdi2290 Thanks for pointing in this direction, I like it! I'm very busy ATM at work and trying to organize a community fork of Famous. If you want to submit a pull request with this working I would review and probably integrate it. I'm curious to see if this approach can make it possible to insert Angular templates and maintain property and event binding in a rendered Famous DOMElement.

**Comment by PatrickJS** [comment]
@steveblue can I ask why you're doing a community fork of Famous? I haven't been up to date on Famous news

**Comment by steveblue** [comment]
Take a look in the famous.org Community Slack #general room August 18th. Many people have felt the company, probably due to poor leadership.

**Comment by trusktr** [comment]
ANgular operates on the HTML present in the DOM right (like ANgular 1.0, where it'd start working off of what exists in DOM)?

If so, I'd rather write famous components that way, in the HTML that goes directly to the DOM, not as Framework style components.

**Comment by trusktr** [comment]
@gdi2290 8 of the core team left Famous. The two main guys that started Framework left.  At least 3 of Engine's WebGL team left. Our main community guy Myles (@thealphanerd) left. This all happened at once.

**Comment by PatrickJS** [comment]
yeah I heard, Angular can work without the DOM and does so in order for it to run in a web worker/server/native

**Comment by the-reality-engineer** [comment]
@trusktr how do you know who left Famous? Do you know why they left?

**Comment by trusktr** [comment]
@speigg I'm not entirely sure. I just know they did. It's been news floating around in Slack, plus a couple friends in Famous had said things were going to shift so they wanted to leave before that happened.

**Comment by PatrickJS** [comment]
ya I hear there is only 6 people left
